### PR TITLE
Manual cherry-pick of 37890 into the `release/7.7`

### PR DESCRIPTION
**Pull request by psealock**
This is a manual cherry-pick of https://github.com/woocommerce/woocommerce/pull/37890 into the `release/7.7` branch.

There was [a conflict](https://github.com/woocommerce/woocommerce/actions/runs/4780637816) with the changes made previously in https://github.com/woocommerce/woocommerce/pull/37808.

@mattsherman @Sidsector9, could you please verify my manual conflict resolution was done correctly?

### CODE REVIEW

File: plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js [hunk 1]
```diff
@@ -1093,6 +1104,7 @@ jQuery( function ( $ ) {
 	$( '#wp-content-media-buttons' )
 		.append( '<span class="woocommerce-help-tip" tabindex="-1"></span>' )
 		.find( '.woocommerce-help-tip' )
+		.attr( 'tabindex', '0' )
```
Comment (mattsherman) [inline plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js:1107]: Not really sure why these needed to come into this PR, but they do not cause harm and those help tips function properly.

File: plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js [hunk 2]
```diff
@@ -1111,6 +1123,7 @@ jQuery( function ( $ ) {
 	$( '#postexcerpt > .postbox-header > .hndle' )
 		.append( '<span class="woocommerce-help-tip"></span>' )
 		.find( '.woocommerce-help-tip' )
+		.attr( 'tabindex', '0' )
```
Comment (mattsherman) [inline plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js:1126]: Not really sure why these needed to come into this PR, but they do not cause harm and those help tips function properly.

**Review by mattsherman — APPROVED** [review]
> There was [a conflict](https://github.com/woocommerce/woocommerce/actions/runs/4780637816) with the changes made previously in https://github.com/woocommerce/woocommerce/pull/37808.
> @mattsherman @Sidsector9, could you please verify my manual conflict resolution was done correctly?

I have verified that the changes from #37890 have been correctly conflict resolved.

I do not understand why two changes from #37808 were pulled into here, as I don't see why they would have been needed (it's Monday morning, maybe I'm not fully awake yet). But, I have confirmed that they do not cause any issues. As #37808 is not part of 7.7 and will be in 7.8, I think this is alright.

Approving.

**Comment by psealock** [comment]
>I do not understand why two changes from https://github.com/woocommerce/woocommerce/pull/37808 were pulled into here

Thanks for the review @mattsherman, and yes I must not be properly caffeinated either because I'm not sure what happened there either. Thanks for confirming it all works properly

**Pull request merged**